tests/network/primary_pod_network.go

```diff
@@ -52,7 +53,7 @@ var _ = SIGDescribe("Primary Pod Network", func() {
 			By("Making sure IP reported on the VMI matches the one on the pod")
 			Expect(vmi.Status.Interfaces[0].IP).To(Equal(vmiPod.Status.PodIP))
 			Expect(vmi.Status.Interfaces[0].IPs).NotTo(BeEmpty())
-			Expect(vmi.Status.Interfaces[0].IPs[0]).To(Equal(vmiPod.Status.PodIP))
+			Expect(reflect.DeepEqual(vmi.Status.Interfaces[0].IPs, vmiPod.Status.PodIPs)).To(BeTrue())
```

Comment:
Use `Equal` as Ram suggested on the other PR.

Reply:
done in the 2 places in this PR, will do there as well

Reply:
```
  Expected
      <[]string | len:1, cap:1>: ["10.244.140.105"]
  to equal
      <[]v1.PodIP | len:2, cap:2>: [
          {IP: "10.244.140.105"},
          {IP: "fd10:244::8c68"},
      ]
```

i before i used reflect.DeepEqual i used a for loop that interate and use ip.IP
because they are not the same type, as each element in the other is v1.PodIP

will change it for a for again